src/gui/src/scriptWidget.cpp

```diff
@@ -115,6 +116,10 @@ ScriptWidget::ScriptWidget(QWidget* parent)
           &ScriptWidget::outputChanged);
   connect(pauser_, &QPushButton::pressed, this, &ScriptWidget::pauserClicked);
   connect(pause_timer_.get(), &QTimer::timeout, this, &ScriptWidget::unpause);
+  connect(report_timer_.get(),
+          &QTimer::timeout,
+          this,
+          &ScriptWidget::flushReportBufferToOuput);
```

Comment:
you should also connect this to `commandFinishedExecuting` to avoid a delay in printing the output.